.circleci/config.yml

```diff
@@ -8,36 +8,30 @@ buildSteps: &buildSteps
   - run:
       name: test
       command: npm test
-  - persist_to_workspace:
-      # Persist all job output, so we can (potentially) use it for deploys
-      root: ../
-      paths:
-        - ./node-*
 
 jobs:
   "node-6":
     docker:
       - image: circleci/node:6
-    working_directory: ~/node-6
     steps: *buildSteps
 
   "node-8":
     docker:
       - image: circleci/node:8
-    working_directory: ~/node-8
     steps: *buildSteps
 
   deploy:
     # For this to work NPM_TOKEN must be set for the account used for publishing
     docker:
       - image: circleci/node:8
     steps:
-      - attach_workspace:
-          at: .
+      - checkout
+      - run:
+        name: install-npm
+        command: npm install
```

Comment:
This isn't indented properly, so it won't work. This actually parses (e.g. with http://yaml-online-parser.appspot.com/) as

```json
        {
          "run": null, 
          "command": "npm install", 
          "name": "install-npm"
        }, 
```

whereas you want

```json

        {
          "run": {
            "command": "npm test", 
            "name": "test"
          }
        }
```

YAML is stupid